doc/tech/glossary.md

```diff
@@ -146,15 +146,24 @@ Terms not defined by Kubernetes:
 - Observability = Observability is a measure of how well internal states of a system can be inferred from knowledge of its external outputs. ([Wikipedia:Observability](https://en.wikipedia.org/wiki/Observability))
 
 <a name="1.7"></a>
-## 1.7 Other Referenced Terminology
+## 1.7 Test Related Terminology
+
+- **Performance Measurement:** The procedure or set of operations having the object of determining a Measured Value or Measurement Result. In this context, Performance Measurements reflect data generated and collected within the cloud infrastructure, that reflects a perfromance aspect of the cloud infrastructure. For example, a count of frames or packets traversing an interface, memory usage information, other resource usage and availability, etc. These data may be instantaneous or accumulated, and made available (i.e. exposed) based on permissions and contexts (e.g., workload vs. infra)
```

Comment:
nitpick: Since using "For example" then "etc." is unnecessary

Reply:
Added period